vidarrtest-regression.json.in

```diff
@@ -394,7 +394,7 @@
            {
             "metrics_calculate": "@CHECKOUT@/tests/calculate.sh",
             "metrics_compare": "@CHECKOUT@/tests/compare.sh",
-            "output_metrics": "/.mounts/labs/gsi/testdata/bclconvert/output_metrics/MiSeqTile240108_ONESAMPLE.metrics",
+            "output_metrics": "/.mounts/labs/gsi/testdata/bclconvert/output_metrics/MiSeqTile240108_ONESAMPLE_D.metrics",
```

Comment:
Is it known what is causing this difference? Just sort order or something like that...?

Reply:
I checked the files and yes, it seems that it is the sort order. The number of the output reads is the same in both cases